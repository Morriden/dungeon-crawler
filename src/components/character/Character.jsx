import React from 'react';
import PropTypes from 'prop-types';

const Character = ({ image }) => (
  <div>
    <img src={image} />
  </div>
);

Character.propTypes = {
  image: PropTypes.string.isRequired
};

export default Character;
