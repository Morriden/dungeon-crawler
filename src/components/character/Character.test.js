import React from 'react';
import { shallow } from 'enzyme';
import Character from './Character';

describe('Character component', () => {
  it('renders Character image', () => {
    const wrapper = shallow(<Character image="im an image" />);
    expect(wrapper).toMatchSnapshot();
  });
});
