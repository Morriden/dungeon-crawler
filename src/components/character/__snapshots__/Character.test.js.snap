// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component renders Character image 1`] = `
<div>
  <img
    src="im an image"
  />
</div>
`;
